### PR TITLE
Full jammy support

### PR DESCRIPTION
**Pull request by ru-fu**

### CODE REVIEW

File: howto/monitor/collect-metrics.md [hunk 1]
```diff
@@ -87,10 +87,10 @@ Complete the following steps to deploy Anbox Cloud with the reference monitoring
      - ['ams:grafana', 'grafana:dashboards']
    machines:
      '0':
-       series: focal
+       series: jammy
```
Comment (ru-fu) [inline howto/monitor/collect-metrics.md:90]: Do the updates in this file make sense?

File: howto/monitor/monitor-status.md [hunk 2]
```diff
@@ -41,13 +41,13 @@ Complete the following steps to deploy Anbox Cloud with the reference monitoring
      - ['nrpe', 'nagios']
    machines:
      '0':
-       series: bionic
+       series: jammy
```
Comment (ru-fu) [inline howto/monitor/monitor-status.md:44]: Do the updates in this file make sense?

File: howto/install/deploy-juju.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ There are differences between the full Anbox Cloud installation and the Anbox Cl
 
 Before you start the installation, ensure that you have the required credentials and prerequisites:
 
-* An Ubuntu 18.04 LTS or 20.04 LTS environment to run the commands (or another operating system that supports snaps - see the [Snapcraft documentation](https://snapcraft.io/docs/installing-snapd)).
+* An Ubuntu 18.04 LTS, 20.04 LTS, or 22.04 LTS environment to run the commands (or another operating system that supports snaps - see the [Snapcraft documentation](https://snapcraft.io/docs/installing-snapd)).
```
Comment (morphis) [inline howto/install/deploy-juju.md:13]: Ah, good point: We have to drop Ubuntu 18.04 here as we no longer support it starting with 1.16

I missed on that, can you add that here or should I provide a separate PR?

File: howto/monitor/monitor-status.md (hunk 2 above)
Comment (morphis) [reply howto/monitor/monitor-status.md:44]: This becomes now even trickier as nagios must stay on bionic. It doesn't support focal or jammy.

Can you leave this one as is for now and I will do a PR next week to clean this up?

File: howto/install/deploy-juju.md (hunk 3 above)
Comment (morphis) [reply howto/install/deploy-juju.md:13]: Can we also declare 22.04 as the recommended version to use?

File: howto/monitor/collect-metrics.md (hunk 1 above)
Comment (morphis) [reply howto/monitor/collect-metrics.md:90]: See my comments below for nagios.

**Review by morphis — CHANGES_REQUESTED** [review]

File: howto/install/deploy-juju.md (hunk 3 above)
Comment (ru-fu) [reply howto/install/deploy-juju.md:13]: Updated.

But we still support the bionic images, correct?
https://anbox-cloud.io/docs/ref/provided-images

File: howto/monitor/monitor-status.md (hunk 2 above)
Comment (ru-fu) [reply howto/monitor/monitor-status.md:44]: I dropped the commit.

**Review by morphis — APPROVED** [review]
LGTM, thanks!

**Pull request merged**